app/src/main/res/values/strings.xml

```diff
@@ -237,6 +237,10 @@
   <string name="export_data_error">Error exporting data</string>
   <string name="exported_data">Data exported to %1$s</string>
 
+  <string name="dtm_move_up">Move Up</string>
```

Comment:
Out of curiosity, what does "dtm" stand for? :)

Reply:
lol 'downloaded translation menu' :)  just needed to make it unique

Reply:
I don't think there is a need for unique names here, make them generic (simple use `move_up`, `move_down` and `delete_translation`?). We might as well re-use them elsewhere in the future.